client/validate.go

```diff
@@ -35,6 +35,7 @@ type validatorParams struct {
 	tag                 string
 	functionType        string
 	validateConcurrency bool
+	envs                string
```

Comment:
It's probably more convenient to make this field a `[]string` and do the `strings.Split` loop once in `main.go` so that all the downstream consumers don't have to parse it again